assets/i18n/ast.json

```diff
@@ -15,8 +15,8 @@
 	"I18N_CREATE_EXPLORATION_UPLOAD": "Xubir",
 	"I18N_CREATE_NO_THANKS": "Non, gracies",
 	"I18N_CREATE_YES_PLEASE": "¡Sí, por favor!",
-	"I18N_DASHBOARD_MY_COLLECTIONS": "Les mios colecciones",
-	"I18N_DASHBOARD_MY_EXPLORATIONS": "Les mios esploraciones",
+	"I18N_DASHBOARD_COLLECTIONS": "Les mios colecciones",
```

Comment:
Hm. Can you go through these and make a best guess of what it would take to remove the "My" part? I'm concerned that we are going to have some stale values here that translate to "My ..." instead of just "...".

See if you can deduce what the correct expression should be based on other translations of things involving "collections" etc. It's an interesting linguistics exercise, and it's probably going to be easier for a translator to fix a minor grammatical error.